qa/src/test/resources/application.yml

```diff
@@ -0,0 +1,17 @@
+camunda:
+  migrator:
+    auto-ddl: true
+    c7:
+      data-source:
+        auto-ddl: true
+        jdbc-url: jdbc:h2:mem:migrator
+        username: sa
+        password: sa
+        driver-class-name: org.h2.Driver
+    c8:
+      data-source:
+        auto-ddl: true
+        jdbc-url: jdbc:h2:mem:migrator
+        username: sa
+        password: sa
+        driver-class-name: org.h2.Driver
```

Comment:
🔧 C8 data source was removed on purpose for Runtime. If we need it for history,  I suggest creating a profile properties file used in the history tests. Same goes for PG and ORCL. If I am not mistaken, it will reduce the connections and thus the resources. Feel free to challenge.

Reply:
I thought extending profiles is possible out of the box in Spring Boot but I couldn't find how. Your solution looks good so let's merge and iterate if need later.